wp-content/themes/haruoshi_origin/archive-works.php

```diff
@@ -13,13 +13,13 @@
 		<?php if ( have_posts() ):?>
 			<ul class="works__article-list">
 				<?php while ( have_posts() ) : the_post(); ?>
-					<li class="works__article">
+					<a class="works__article" href="<?php the_permalink() ?>">
```

Comment:
ulタグの子要素は必ずliです。
リンクを付けたい場合はliの中にaタグを追加します。
```suggestion
					<li class="works__article">
						<a href="<?php the_permalink() ?>">

```

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/3fa6155e234755cfc5f6a20032049983f0cc9c24